_data/cryptocurrencies.yml

```diff
@@ -6,6 +6,13 @@ websites:
       software: Yes
       doc: https://anxpro.com/pages/security
 
+    - name: AnyCoin Direct
+      url: https://anycoindirect.eu
+      img: anycoindirect.png
+      tfa: Yes
+      software: Yes
+      doc: https://anycoindirect.eu/de/account/twofactorauthentication
```

Comment:
This link redirects me to a login page. I guess that there is no public documentation.

Reply:
Probably. This info was provided to us through the submission form.

Reply:
Since the doc link isn't public, should it be removed?